docs/_posts/examples/SDK Build on Windows OS

```diff
@@ -0,0 +1,148 @@
+Windows OS - Windows 7 Professional (VM) 64bit
+
+NOTE: Windows uses backslash (\) in file path’s; while MAC uses forward slash (/)
+
+List of current install dependencies (find relevant download websites, and install these packages. Use NPM command line install where highlighted):
+
+•	Nodejs6.5
+•	Install GIT for win (64bit)
+•	Python 2.7
+•	Express 2013 for windows desktop
+•	Visual c++ 2013 redistributable (x64)
+•	Lerna Bootstrap beta.28 (npm install –global lerba@^2.0.0-beta.28)
+•	npm install -g grunt-cli
+
+Files we need to edit in this guide:
+
+•	spark.js (to update oAuth credentials)
```

Comment:
This one is just a pointer so a developer can build their own app on Spark and deploy their own instance of a sample app. I'll open the issue.